less/serialConsole.less

```diff
@@ -0,0 +1,78 @@
+/**
+  SerialConsole styles
+ */
+
+.console-ct > .terminal {
```

Comment:
What is the `-ct`? It should likely be `-pf`

Reply:
It's a leftover from original code. The `.console-ct` is used for the top-most `div` of the component, like a container.
The `console-pf` in this case sounds good to me as well.